modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -52,6 +54,36 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
 sudo chmod +x rtfctl
 ----
 
+For Runtime Fabric on appliance:
+
+. Download the `rtfctl` command-line utility:
++
+`rtfctl` is supported on Windows, MacOS (Darwin), and Linux. Download this utility using the URLs below:
```

Comment:
You may want to use "following" instead of "below". I think "below" is not accessible.